api/Clipboard.json

```diff
@@ -82,15 +82,7 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": "90",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.events.asyncClipboard.read",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "Firefox only supports reading the clipboard in browser extensions, using the <code>\"clipboardRead\"</code> extension permission."
+              "version_added": "preview"
```

Comment:
FYI The note is removed because `dom.events.asyncClipboard.readText` (which replaced `dom.events.asyncClipboard.read`) pref has comment _Control whether `navigator.clipboard.readText()` is exposed to content._